build-scripts/hack/update-component-versions.py

```diff
@@ -27,7 +27,7 @@
 DIR = Path(__file__).absolute().parent
 SNAPCRAFT = DIR.parent.parent / "snap/snapcraft.yaml"
 COMPONENTS = DIR.parent / "components"
-CHARTS = DIR.parent.parent / "k8s" / "manifests" / "charts"
+CHARTS = DIR.parent.parent / "src" / "k8s" / "pkg" / "k8sd" / "features" / "coredns" / "charts"
```

Comment:
I believe this is not quite right, it seems like we should update the references to this variable with appropriate paths for components, because I see there is a call for metallb, contour and so on.